docs/modules/ROOT/pages/python-client-getting-started.adoc

```diff
@@ -0,0 +1,195 @@
+* Docker
+* Python 3.6+
+
+== Start a Hazelcast Member
+
+We will use the 5.3 version of Hazelcast for this tutorial.
+
+In this tutorial, we will use https://hazelcast.com/products/viridian/[Hazelcast Viridian Cloud].
+
+1. Sign up for a Hazelcast Viridian Cloud account (Free accounts are available).
+2. Log in to your Hazelcast Viridian Cloud account and navigate to your https://viridian.hazelcast.com[cluster list].
+3. Create a new cluster or start an existing one.
+
+
+== Install Hazelcast Python Client
+
+1. Identify the Hazelcast Viridian Cloud https://viridian.hazelcast.com[cluster] you want to connect to and choose Connect Cluster.
+2. Use the quick connection guide to download the preconfigured Python sample client.
+3. Follow the instructions on the quick connection guide to start using your cluster.
+
+To understand and use the client, review the https://hazelcast.readthedocs.io/en/stable/client.html#hazelcast.client.HazelcastClient[Python API documentation] to better understand what is possible.
+
+== Starting the Python Client
+
+The following section creates and starts a new Hazelcast client with default configuration.
+
+[source,python]
+----
+import hazelcast
+
+client = hazelcast.HazelcastClient()
+
+client.shutdown()
+----
+
+== Use Map
+
+A Hazelcast map is a distributed key-value store, similar to Python dictionary. You can store key-value pairs in a map.
+In the following example, we will work with map entries where the keys are ids and the values are defined as an object representing a city.
+
+[source,python]
+----
+client = hazelcast.HazelcastClient()
+
+cities_map = client.get_map('cities').blocking()
+cities_map.set(1, "London")
+cities_map.set(2, "New York")
+cities_map.set(3, "Tokyo")
+
+entries = cities_map.entry_set()
+
+for key, value in entries:
+    print(f"{key} -> {value}")
+
+client.shutdown()
+----
+
+The output of this snippet is given below:
+
+[source,bash]
+----
+2 -> New York
+1 -> London
+3 -> Tokyo
+----
+
+Following line returns a map proxy object for the `cities` map:
+
+[source, python]
+----
+cities_map = client.get_map('cities').blocking()
+----
+
+If `cities` doesn't exist, it will be automatically created. All the clients connected to the same cluster will have access to the same map.
+
+You may wonder why we have used `blocking()` method over the `get_map()`. This returns a version of this proxy with only blocking
+(sync) method calls, which is better for getting started. For async calls, please check our https://hazelcast.readthedocs.io/en/stable/#usage[API documentation].
+
+With these lines, client adds data to `cities` map. The first parameter is the key of the entry, the second one is the value as mentioned above.
+
+[source, python]
+----
+cities_map.set(1, "London")
+cities_map.set(2, "New York")
+cities_map.set(3, "Tokyo")
+----
+
+Then, we get the values with the `entry_set()` method and iterate over the set.
+
+[source, python]
+----
+entries = cities_map.entry_set()
```

Comment:
is there a specific reason you chose entry_set instead of the map.get API as in [nodejs getting started](git@github.com:hazelcast/hazelcast-python-client.git)?

Reply:
All our guides have the version specified - this is a consistency copy from C# and Java.

Suggest fixing them ALL after the first cut is made available.